client/my-sites/checkout/checkout/pay-button.jsx

```diff
@@ -142,17 +141,18 @@ class PayButton extends React.Component {
 	};
 
 	render() {
-		var buttonState = this.buttonState();
+		const buttonState = this.buttonState();
 
 		return (
 			<span className="pay-button">
-				<button
+				<Button
 					type="submit"
 					className="button is-primary button-pay pay-button__button"
+					busy={ buttonState.disabled }
 					disabled={ buttonState.disabled }
```

Comment:
I don't think we need this any longer. (Actually, I don't think this prop is even used for `Button`.)

Reply:
It's passed down to the `button` component.